product_manufacturer/views/product_manufacturer_view.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" ?>
+<odoo>
+
+    <record id="product_template_manufacturer_form_view" model="ir.ui.view">
+        <field name="name">product.template.form - product_manufacturer</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view"/>
+        <field name="arch" type="xml">
+            <page name="general_information" position="inside">
+                <group name="manufacturer" string="Manufacturer">
+                    <group>
+                        <field name="manufacturer" context="{'default_supplier':True, 'default_customer':False}"/>
```

Comment:
It seems this context is not working, I can select customers and suppliers now.

Reply:
Right, odoo behaves differently in this regard now: it always shows everything (suppliers and customers), but orders them by putting suppliers (or customers) first. The new context key to use is `{'res_partner_search_mode': 'supplier'}`. I fixed this.

Thanks for the review!

Reply:
Thanks! Works now.